tests/test_conda_env.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+import importlib
```

Comment:
whoops missing future import

```suggestion
from __future__ import annotations

import importlib
```